src/common/components/table/filters/table-filters-utils.tsx

```diff
@@ -60,6 +60,25 @@ export const useDateFilterHandler = (useShallow: boolean = true) => {
   };
```

Comment:
I think adding a unit test suite for this file would be really valuable to help catch edge cases and give us confidence updating it in future.